src/util-logopenfile.c

```diff
@@ -64,8 +65,10 @@ SCLogOpenUnixSocketFp(const char *path, int sock_type)
     return ret;
 
 err:
-    SCLogError(SC_ERR_SOCKET, "Error connecting to socket \"%s\": %s",
-               path, strerror(errno));
+    if (log_err)
+        SCLogError(SC_ERR_SOCKET,
```

Comment:
In new code I'm really doing errors only for unrecoverable events, while warnings are used for recoverable events. So here a warning would make sense.